.evergreen/config.yml

```diff
@@ -192,7 +192,7 @@ functions:
     - command: shell.exec
       params:
         script: |
-          DRIVERS_TOOLS="${DRIVERS_TOOLS}" sh ${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/build-mongohouse-local.sh
+          VARIANT=debian11 DRIVERS_TOOLS="${DRIVERS_TOOLS}" sh ${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/build-mongohouse-local.sh
```

Comment:
I don't see `VARIANT` referenced in [build-mongohouse-local.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh). What is this used for?

If this is necessary, this approach seems fragile because we're two levels removed from the `run_on: debian11` in the `atlas-data-lake-test` matrix definition, which invokes the `test-atlas-data-lake` task that in turn calls this function.

Reply:
> I don't see `VARIANT` referenced in [build-mongohouse-local.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/atlas_data_lake/build-mongohouse-local.sh). What is this used for?

Took me a while. The `VARIANT` environment variable is read in the [mongohouse build script](https://github.com/10gen/mongohouse/blob/9487b1aea0534c5d6f8923366b0f3dd9b269c5e3/cmd/buildscript/internal/config.go#L157). It is then used in the [`getDesiredTarget` function](https://github.com/10gen/mongohouse/blob/9487b1aea0534c5d6f8923366b0f3dd9b269c5e3/cmd/buildscript/internal/config.go#L1422). The value returned from there is later used to [download the correct MongoDB package](https://github.com/10gen/mongohouse/blob/9487b1aea0534c5d6f8923366b0f3dd9b269c5e3/cmd/buildscript/internal/config.go#L1481) and return the correct path to the binaries.

> If this is necessary, this approach seems fragile because we're two levels removed from the `run_on: debian11` in the `atlas-data-lake-test` matrix definition, which invokes the `test-atlas-data-lake` task that in turn calls this function.

That's a good point. I've changed this to add a `VARIANT` extension where we define the build variant, setting the environment variable from that expansion. That puts the definition of the build variant to where we also specify the OS the tests run on.